examples/network-manager-app/network-manager-common/network-manager-app.zap

```diff
@@ -3420,7 +3420,7 @@
               "storageOption": "External",
               "singleton": 0,
               "bounded": 0,
-              "defaultValue": "",
+              "defaultValue": null,
```

Comment:
are these "" -> null changes intentional?

Reply:
No, they are changed by zap automatically after I modified the zap file.